iac-common/src/main/java/org/sonar/iac/common/extension/visitors/InputFileContext.java

```diff
@@ -56,18 +58,19 @@ public void reportIssue(RuleKey ruleKey, @Nullable TextRange textRange, String m
       NewIssueLocation issueLocation = issue.newLocation().on(inputFile).message(message);
 
       if (textRange != null && TextRanges.isValidAndNotEmpty(textRange)) {
-        issueLocation.at(toInputFileRange(textRange));
+        issueLocation.at(toInputFileRange(inputFile, textRange));
       }
 
       issue.forRule(ruleKey).at(issueLocation);
 
       secondaryLocations.stream()
         .filter(location -> location != null && TextRanges.isValidAndNotEmpty(location.textRange))
```

Comment:
Nitpick: both lambdas operate with same objects, but arguments are called differently (`location` and `secondary`)